### PR TITLE
Add Optimism Tables to Dune Explorer UI

### PR DESCRIPTION
**Pull request by chuxinh**
Brief comments on the purpose of your changes:

Adding recently added and existing OP tables to front end explorer UI. 
Also updated the alias of `transfers_optimism_eth` to `eth`. Want it to show up as `eth` as table name instead of the following
image

Hope this won't break anything 🙏 

*For Dune Engine V2*
I've checked that:

* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`

When you are ready for a review, tag duneanalytics/data-experience. We will re-open your forked pull request as an internal pull request. Then your spells will run in dbt and the logs will be avaiable in Github Actions DBT Slim CI. This job will only run the models and tests changed by your PR compared to the production project.

### CODE REVIEW

File: macros/alter_table_properties.sql [hunk 1]
```diff
@@ -550,6 +550,43 @@ ALTER TABLE airdrop_optimism.addresses SET TBLPROPERTIES ('dune.public'='true',
...
+ALTER TABLE transfers_optimism.eth SET TBLPROPERTIES ('dune.public'='true',
+                                                'dune.data_explorer.blockchains'='["optimism"]',
+                                                'dune.data_explorer.category'='abstraction',
+                                                'dune.data_explorer.abstraction.type'='sector',
+                                                'dune.data_explorer.abstraction.name'='transfers',
+                                                'dune.data_explorer.contributors'='["msilb7", "chuxinh"]');
+{% endset %}
+
+{% set uniswap_v3_optimism_pools %}
+ALTER TABLE uniswap_v3_optimism.pools SET TBLPROPERTIES ('dune.public'='true',
```
Comment (jeff-dude) [inline macros/alter_table_properties.sql:563]: from what i can tell in the dbt project file (defaults to view) and model itself (no override in config), this object is a view, not table. if that's expected, this should be updated to ALTER VIEW

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Comment by jeff-dude** [comment]
changing the name to 'eth' is fine, it'll create a new view and the old one will linger. we will need to manually clean that up post-deploy

**Comment by chuxinh** [comment]
Thanks @jeff-dude updated everything here.
If you can let me know how i can bring up those csv files related to OVM1 I'd like to add them here too. Or a separate PR :D

**Comment by jeff-dude** [comment]
> s related to OVM1 I'd like to add them here too. Or a sepa

i think this relates to what we're talking about in TG, right? let's follow up there

**Comment by jeff-dude** [comment]
> i think this relates to what we're talking about in TG, right? let's follow up there



> Thanks @jeff-dude updated everything here. If you can let me know how i can bring up those csv files related to OVM1 I'd like to add them here too. Or a separate PR :D

i think this relates to what we're talking about in TG, right? let's follow up there

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**